app/kumactl/cmd/install/install_gateway.go

```diff
@@ -0,0 +1,68 @@
+package install
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	install_context "github.com/kumahq/kuma/app/kumactl/cmd/install/context"
+	kumactl_data "github.com/kumahq/kuma/app/kumactl/data"
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/data"
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/k8s"
+)
+
+type gatewayTemplateArgs struct {
+	Namespace string
+}
+
+func newInstallGatewayCmd(ctx *install_context.InstallGatewayContext) *cobra.Command {
+	args := ctx.Args
+	cmd := &cobra.Command{
+		Use:   "gateway",
+		Short: "Install ingress gateway on Kubernetes",
+		Long: `Install ingress gateway on Kubernetes in a 'kuma-gateway' namespace.
+This command requires that the KUBECONFIG environment is set`,
```

Comment:
Why KUBECONFIG is required? Do we require a connection to the Kubernetes cluster?

Reply:
Not necessary. I was originally modeling after the DNS installer which requires it, before realizing that this would be much simpler, and then also copied from here to the subsequent PR. Will remove both cases.